server/README.md

```diff
@@ -0,0 +1 @@
+## PandasAI Server
```

Comment:
Would be interesting a quick overview on how to run the server and a brief overall description